build/ferretdb-bw/Dockerfile

```diff
@@ -6,10 +6,13 @@ FROM --platform=linux/amd64 ghcr.io/ferretdb/postgres-documentdb-dev:17-ferretdb
 
 RUN --mount=type=cache,target=/var/cache/apt <<EOF
 
+apt install -y curl
+
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
 
-curl -L https://pgp.mongodb.com/server-7.0.asc | apt-key add -
-echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 main" | tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+curl -L https://pgp.mongodb.com/server-7.0.asc | \
+gpg --dearmor --yes --output /usr/share/keyrings/mongodb-bookworm.gpg
```

Comment:
We generally indent lines that follow `\` with a tab. Please do so in that and other changed files.

Reply:
Do you have an example? I can't find one in the repository and VS Code wants to convert `<tab>` into two spaces. I can't tell if you want an actual tab,  two spaces, four spaces, or eight spaces.